### ISSUE TITLE
java compiler error (class not found) when using scala dependency

### ISSUE THREAD

**Issue by codejanovic** [manifold-systems/manifold#461]
**Describe the bug**
When using [Akka](https://github.com/akka/akka) within a project that configured manifold, the compiler throws the following error:
```
An exception has occurred in the compiler (17.0.7). Please file a bug against the Java compiler via the Java bug reporting page (https://bugreport.java.com/) after checking the Bug Database (https://bugs.java.com/) for duplicates. Include your program, the following diagnostic, and the parameters passed to the Java compiler in your report. Thank you.
com.sun.tools.javac.code.Symbol$CompletionFailure: class file for akka.actor.CoordinatedShutdown$tasks$$TaskDefinition$ not found
```

**To Reproduce**
I forked the manifold-sample-project and added the akka dependency + a sample class using akka to reproduce the error: https://github.com/codejanovic/manifold-sample-project 

**Expected behavior**
I expected the project to compile correctly.

**Screenshots**
- Build success without akka dependency: [here](https://github.com/codejanovic/manifold-sample-project/actions/runs/5435814975/jobs/9885216321)
- Build failed with akka dependency: [here](https://github.com/codejanovic/manifold-sample-project/actions/runs/5435818146/jobs/9885221065)

**Desktop (please complete the following information):**
- OS Type & Version: MacOS / Github Action llinux
- Java/JDK version: 17 
- IDE version (IntelliJ IDEA or Android Studio): 2023.1.3
- Manifold version: 2023.1.10
- Manifold IntelliJ plugin version: 2023.1.10

**Additional context**
-

**Stack trace**
https://github.com/codejanovic/manifold-sample-project/actions/runs/5435818146/jobs/9885221065

**Comment by rsmckinney** [comment]
That is a confusing naming convention with akka/scala -- Name vs. Name$ etc. I'm not sure what is going on exactly, but it appears the `akka.actor.CoordinatedShutdown$tasks$$TaskDefinition$` class is indeed missing from the jar or is otherwise not available, however it manifests. Perhaps there is another akka related jar dependency that defines it?

**Comment by codejanovic** [comment]
You are right, the type `akka.actor.CoordinatedShutdown$tasks$$TaskDefinition$` only exists within the scala code of akka, but i am also unable to find it within the jar. I will try to find out what is happening here and report back! Thanks for the fast response

**Comment by codejanovic** [comment]
It looks like the classes are not referenced in IntelliJ, but present in the jar:

<img width="877" alt="image" src="https://github.com/manifold-systems/manifold/assets/7228633/dbca9d5a-5894-4bd8-b313-40cee8bde78c">

<img width="426" alt="image" src="https://github.com/manifold-systems/manifold/assets/7228633/f3affeba-c7a7-49ac-8d46-93f5dac399fc">

**Comment by rsmckinney** [comment]
The missing type has a pair of `$$`:

akka.actor.CoordinatedShutdown$tasks$$TaskDefinition$

**Comment by codejanovic** [comment]
Is there anything i can help investigate with? Although the compiler complains about `...$tasks$$TaskDefinition$`, it works lilke a charme when manifold is not configured.

**Comment by rsmckinney** [comment]
> it works lilke a charme when manifold is not configured

Good point. It must be something on my end. I’ll have another look at this.

**Comment by codejanovic** [comment]
Just checked that i'm not lying:

**manifold disabled** -> compiles: [commit](https://github.com/codejanovic/manifold-sample-project/commit/e517049de01a014e7b8cb047d6bd2d0d181c3d99)
**manifold enabled** -> wont compile: [commit](https://github.com/codejanovic/manifold-sample-project/commit/32900ae8561ff5ced8e5a03b86a6599768db8543)

**Comment by LinSanmu** [comment]
Has this bug been fixed?

**Comment by rsmckinney** [comment]
Hi @LinSanmu. Sorry, no it’s an issue that unfortunately fell by the wayside. I’ll put it back on my stack. Are you experiencing the same problem?
